components/external_child_process/child_monitor.cc

```diff
@@ -0,0 +1,186 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/external_child_process/child_monitor.h"
+
+#if defined(OS_POSIX)
+#include <errno.h>
+#include <fcntl.h>
+#include <signal.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+#elif defined(OS_WIN)
+#include <windows.h>
+#endif
+
+#include <utility>
+
+#include "base/bind_post_task.h"
+#include "base/logging.h"
+#include "base/process/kill.h"
+#include "base/single_thread_task_runner.h"
+#include "base/task/post_task.h"
+#include "base/threading/thread_task_runner_handle.h"
```

Comment:
`sequenced_task_runner_handle.h`

Reply:
fixed in 76857501ecbfb2f31ff1023ec197de6d90d62618